sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -2221,6 +2221,7 @@ class AstBuilder(conf: SQLConf) extends SqlBaseBaseVisitor[AnyRef] with Logging
       case ("decimal" | "dec" | "numeric", precision :: scale :: Nil) =>
         DecimalType(precision.getText.toInt, scale.getText.toInt)
       case ("void", Nil) => NullType
+      case ("null", Nil) => NullType
```

Comment:
this is for compatible, like`from_csv`.

Reply:
In which case do we need it? If we have this PR, the valid string for null type should be `void`. Before this PR, it wasn't supported before.

Reply:
Forgot it, we not support parse null before.